### ISSUE TITLE
Remove setup.cfg

### ISSUE THREAD

**Issue by VannTen** [thoth-station/adviser#2421]
Revert workaround introduced in 6bf28d50ae4efb22abc8e2eae359d6b28e58f58b once s2i base image properly detect pyproject.toml
/kind cleanup

**Comment by ishan301** [comment]
Hi, I am a new contributor and I would like to contribute to this project. So, I just need to remove the setup.cfg file and raise a PR right?

**Comment by VannTen** [comment]
Yes. But as noted this currently does not work with our base s2i image, so we'll only change it once that's resolved.
